### ISSUE TITLE
Dos repositorios distintos para el mismo curso

### ISSUE THREAD

**Issue by IlyaOrson** [ata0k/F-sica-Computacional-2018-1#1]
¡Hola! Al parecer David está subiendo los notebooks en [otro repositorio](https://github.com/dpsanders/FisicaComputacional2018_1). Esto es algo confuso, ¿cuál será el oficial?

**Comment by ata0k** [comment]
Sí, ya lo noté!! Al parecer no puso atención en que ya existía uno... y ya subió más cosas él que yo, así que supongo que quedará el suyo como el buen repositorio. Mudaré lo de aquí a su repositorio.
